integration_test/third_party_apps_data/applications/flink/metadata.yaml

```diff
@@ -0,0 +1,60 @@
+public_url: "https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/flink"
+app_url: https://flink.apache.org/
+short_name: Flink
+long_name: Apache Flink
+logo_path: ""
+description: The Apache Flink integration collects client, jobmanager and taskmanager
+  logs and parses them into a JSON payload. The result includes fields for source,
+  level, and message.
+supported_app_version: ["1.13.6", "1.14.4"]
+configure_integration: ""
+expected_metrics:
```

Comment:
This list seems incomplete? There should be more expected metrics right?

Reply:
updated. added rest of the metrics in.